webpages/styles/colors.css

```diff
@@ -47,7 +47,7 @@
   --modal-overlay: rgba(0, 0, 0, 0.65);
   --hover-highlight: rgba(255, 255, 255, 0.05);
   --hover-highlight-reduced: rgba(255, 255, 255, 0.02);
-  --hover-darken: rgba(0, 0, 0, 0.1);
+  --hover-darken: rgba(0, 0, 0, 0.2);
```

Comment:
I don't think this change is necessary. The color doesn't have much contrast with the background either way, and it doesn't need to - the orange border and the font weight change are enough to indicate selection.